src/components/FilterControls/MultiSelect/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react'
+import OutlinedInput from '@mui/material/OutlinedInput'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import FormControl from '@mui/material/FormControl'
+import Select, { SelectChangeEvent } from '@mui/material/Select'
+import Checkbox from '@mui/material/Checkbox'
+import ListItemText from '@mui/material/ListItemText'
+
+interface MultiSelectProps {
+  options: string[]
+  defaultOptions: string[]
+  onClose: (selectedOptions: string[]) => void
+  label: string
```

Comment:
Label should be constant through the lifetime of your app, so we can inline this value instead of having it as a prop.

Reply:
Ah, but multiline is a reusable component. I did that so I  wouldn't have to repeat the same MUI code twice. In this case, label does need to be a prop so that the genres and years dropdowns are different.